workspaces/diff-engine/scripts/install.js

```diff
@@ -9,6 +8,8 @@ dotenv.config({
   path: envPath,
 });
 
+const { install } = require('../lib');
```

Comment:
@JaapRood is this ok (changing the behavior by changing the require order so the env has been loaded) or would you do it a different way?

Reply:
In general I'd say don't rely on the order of requires, but at the same time it's obvious to me that `dotenv` is what I would consider part of the program's "prelude". So yeah, it's okay, I wouldn't know of a better way (if any at all).